utils/src/treev2/quickPickWizard/experiences/findByIdExperience.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as types from '../../../../index';
+import * as vscode from 'vscode';
+import { getLastNode } from '../QuickPickWizardContext';
+import { NoResourceFoundError } from '../../../errors';
+import { FindByIdQuickPickStep } from '../FindByIdQuickPickStep';
+import { isWrapper } from '../../../registerCommandWithTreeNodeUnwrapping';
+import { AzureWizard } from '../../../wizard/AzureWizard';
+import { ResourceGroupsItem } from '../quickPickAzureResource/tempTypes';
+
+export async function findByIdExperience<TPick extends types.FindableByIdTreeNode>(context: types.IActionContext, tdp: vscode.TreeDataProvider<ResourceGroupsItem>, id: string): Promise<TPick> {

```

Comment:
I'm thinking about removing the `findByIdExperience` altogether for a couple of reasons:

1. We aren't accounting for the custom `isAncestorOf` logic that needs to take place to find items under grouping items in the application resource tree.
2. We only need this for compatibility with the v1.5 extensions, so it makes sense for find to be a custom method inside RGs. Fully converted extensions shouldn't use find to reach into other extensions anymore.